[Bug] import javax.validation config from http://easyyapi.com/framwork/javax_validation_ultimate.html ，throw more  exception！！！

at com.itangcent.intellij.config.rule.StringRule$filterWith$1.invoke(StringRule.kt:3)
	at com.itangcent.intellij.config.rule.StringRule$Companion$of$1.compute(StringRule.kt:18)
	at com.itangcent.intellij.config.rule.StringRule$Companion$of$1.compute(StringRule.kt:16)
	at com.itangcent.intellij.config.rule.DefaultRuleComputer$RuleChainImpl.compute(DefaultRuleComputer.kt:74)
	at com.itangcent.idea.plugin.rule.MyStringRuleMode$LIST$compute$1.apply(MyStringRuleMode.kt:15)
	at com.itangcent.idea.plugin.rule.MyStringRuleMode$LIST$compute$1.apply(MyStringRuleMode.kt:11)
	at java.base/java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:195)
	at java.base/java.util.Iterator.forEachRemaining(Iterator.java:133)
	at java.base/java.util.Spliterators$IteratorSpliterator.forEachRemaining(Spliterators.java:1801)
	at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:484)
	at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:474)
	at java.base/java.util.stream.ReduceOps$ReduceOp.evaluateSequential(ReduceOps.java:913)
	at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
	at java.base/java.util.stream.ReferencePipeline.collect(ReferencePipeline.java:578)
	at kotlin.streams.jdk8.StreamsKt.toList(Streams.kt:72)
	at com.itangcent.idea.plugin.rule.MyStringRuleMode$LIST.compute(MyStringRuleMode.kt:17)
	at com.itangcent.intellij.config.rule.DefaultRuleComputer.doComputer(DefaultRuleComputer.kt:55)
	at com.itangcent.intellij.config.rule.DefaultRuleComputer$computer$1.invoke(DefaultRuleComputer.kt:31)
	at com.itangcent.intellij.config.rule.DefaultRuleComputer$computer$1.invoke(DefaultRuleComputer.kt:10)
	at com.itangcent.idea.utils.RuleComputeListenerRegistry.computerListener(RuleComputeListenerRegistry.kt:34)
	at com.itangcent.idea.utils.RuleComputeListenerRegistry.access$computerListener(RuleComputeListenerRegistry.kt:11)
	at com.itangcent.idea.utils.RuleComputeListenerRegistry$computerListener$1.invoke(RuleComputeListenerRegistry.kt:31)
	at com.itangcent.idea.utils.RuleComputeListenerRegistry$computerListener$1.invoke(RuleComputeListenerRegistry.kt:11)
	at com.itangcent.idea.utils.ContextualPsiClassHelper$InnerComputeListener.computer(ContextualPsiClassHelper.kt:254)
	at com.itangcent.idea.utils.RuleComputeListenerRegistry.computerListener(RuleComputeListenerRegistry.kt:30)
	at com.itangcent.idea.utils.RuleComputeListenerRegistry.computer(RuleComputeListenerRegistry.kt:24)
	at com.itangcent.intellij.config.rule.DefaultRuleComputer.computer(DefaultRuleComputer.kt:29)
	at com.itangcent.intellij.config.rule.RuleComputer$DefaultImpls.computer(RuleComputer.kt:16)
	at com.itangcent.intellij.config.rule.DefaultRuleComputer.computer(DefaultRuleComputer.kt:10)
	at com.itangcent.intellij.config.rule.RuleComputerKt.computer(RuleComputer.kt:27)
	at com.itangcent.idea.plugin.api.export.yapi.YapiPsiClassHelper.afterParseFieldOrMethod(YapiPsiClassHelper.kt:70)
	at com.itangcent.intellij.psi.AbstractPsiClassHelper$doGetFields$2.invoke(AbstractPsiClassHelper.kt:585)
	at com.itangcent.intellij.psi.AbstractPsiClassHelper$doGetFields$2.invoke(AbstractPsiClassHelper.kt:30)
	at com.itangcent.intellij.psi.AbstractPsiClassHelper.foreachField(AbstractPsiClassHelper.kt:624)
	at com.itangcent.intellij.psi.AbstractPsiClassHelper.doGetFields(AbstractPsiClassHelper.kt:565)
	at com.itangcent.intellij.psi.AbstractPsiClassHelper.doGetTypeObject(AbstractPsiClassHelper.kt:521)
	at com.itangcent.intellij.psi.DefaultPsiClassHelper.doGetTypeObject(DefaultPsiClassHelper.kt:223)
	at com.itangcent.intellij.psi.AbstractPsiClassHelper.doGetTypeObject(AbstractPsiClassHelper.kt:282)
	at com.itangcent.intellij.psi.AbstractPsiClassHelper.getTypeObject(AbstractPsiClassHelper.kt:134)
	at com.itangcent.idea.plugin.api.export.core.RequestClassExporter.parseResponseBody(RequestClassExporter.kt:796)
	at com.itangcent.idea.plugin.api.export.core.RequestClassExporter.processResponse(RequestClassExporter.kt:354)
	at com.itangcent.idea.plugin.api.export.core.RequestClassExporter.exportMethodApi(RequestClassExporter.kt:197)
	at com.itangcent.idea.plugin.api.export.core.RequestClassExporter.access$exportMethodApi(RequestClassExporter.kt:46)
	at com.itangcent.idea.plugin.api.export.core.RequestClassExporter$doExport$1$1.invoke(RequestClassExporter.kt:150)
	at com.itangcent.idea.plugin.api.export.core.RequestClassExporter$doExport$1$1.invoke(RequestClassExporter.kt:46)
	at com.itangcent.idea.plugin.api.ClassApiExporterHelper$foreachMethod$1$1$1.invoke(ClassApiExporterHelper.kt:147)
	at com.itangcent.idea.plugin.api.ClassApiExporterHelper$foreachMethod$1$1$1.invoke(ClassApiExporterHelper.kt:31)
	at com.itangcent.intellij.context.ActionContext$callInReadUI$1$1.invoke(ActionContext.kt:461)
	at com.itangcent.common.concurrent.ValueHolder.compute(Holder.kt:100)
	at com.itangcent.intellij.context.ActionContext$callInReadUI$1.run(ActionContext.kt:459)
	at com.intellij.openapi.application.ReadAction.lambda$run$1(ReadAction.java:52)
	at com.intellij.openapi.application.impl.ApplicationImpl.runReadAction(ApplicationImpl.java:865)
	at com.intellij.openapi.application.ReadAction.compute(ReadAction.java:61)
	at com.intellij.openapi.application.ReadAction.run(ReadAction.java:51)
	at com.itangcent.intellij.context.ActionContext.callInReadUI(ActionContext.kt:452)
	at com.itangcent.idea.plugin.api.ClassApiExporterHelper$foreachMethod$1$1.invoke(ClassApiExporterHelper.kt:146)
	at com.itangcent.idea.plugin.api.ClassApiExporterHelper$foreachMethod$1$1.invoke(ClassApiExporterHelper.kt:31)
	at com.itangcent.intellij.context.ActionContext$runAsync$2.run(ActionContext.kt:220)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:829)
Caused by: org.codehaus.groovy.control.MultipleCompilationErrorsException: startup failed:
Script664.groovy: 1: unexpected token: . @ line 1, column 53.
   ation.constraints.Positive")&&.contains(
                                 ^

--- a/third/javax.validation.config
+++ b/third/javax.validation.config
@@ -8,6 +8,7 @@ field.required=@javax.validation.constraints.NotNull
 param.required=@javax.validation.constraints.NotEmpty
 field.required=@javax.validation.constraints.NotEmpty
 
+###set resolveMulti = first
 # define var
 number_min=-9999
 number_max=9999
@@ -134,3 +135,5 @@ field.advanced[@javax.validation.constraints.Size]=groovy:```
     }
     return advanced
 ```
+
+###set resolveMulti = error

